pre-cache/copy-env.sh

```diff
@@ -22,3 +25,8 @@ cp /host/etc/containers/registries.conf /etc/containers/registries.conf || true
 cp -a /host/etc/docker /etc/ || true
 cp -a /host/etc/pki/ca-trust/source/anchors /etc/pki/ca-trust/source/ && update-ca-trust || true
 cp -a /host/etc/pki/ca-trust/extracted/pem /etc/pki/ca-trust/extracted/ || true
+[ ! -d "$podman_pull_path" ] && mkdir -p "$podman_pull_path"
+cp $pull_secret_path "$podman_pull_path/auth.json"
+[ ! -d "$docker_pull_path" ] && mkdir -p "$docker_pull_path"
+cp $pull_secret_path "$podman_pull_path/config.json"
```

Comment:
Is this meant to be $docker_pull_path? It looks odd to have it after the docker_pull_path dir check otherwise